server/reportConfiguration/types/FINDS.ts

```diff
@@ -108,7 +108,7 @@ const FINDS: IncidentTypeConfiguration = {
           id: '218694',
           code: 'HANDHELD METAL DETECTOR (HMD) WAND',
           active: true,
-          label: 'HANDHELD METAL DETECTOR (HMD) WAND',
+          label: 'Handheld metal detector (hmd) wand',
```

Comment:
maybe HMD should be an acronym

Reply:
Ah! I thought I added that one, I'll add with the other words mentioned 👍

Reply:
Updated here: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/287/commits/790cdaf0d52e8865428c4d05bd127067f162acdf